resteasy-genericresponse/src/test/scala/com/cerner/beadledom/resteasy/BuiltGenericResponseSpec.scala

```diff
@@ -436,5 +440,30 @@ class BuiltGenericResponseSpec
         verify(rawResponse).getHeaderString("header")
       }
     }
+
+    describe("#toString") {
+      it("returns String with all fields included") {
+        val rawResponse = mock[Response]
+        when(rawResponse.getStatus).thenReturn(200)
+        when(rawResponse.getMediaType).thenReturn(MediaType.APPLICATION_JSON_TYPE)
+        when(rawResponse.getDate).thenReturn(new Date(0L))
```

Comment:
Should be fixed now, https://github.com/cerner/beadledom/pull/28/commits/bd07cae96b21c4610e955327b3f221aebbe23bd8